tests/test_D2_model_constraints.py

```diff
@@ -108,21 +116,17 @@ def test_prepare_constraint_minimal_renewable_share():
         },
         OEMOF_BUSSES: {electricity: electricity, fuel: fuel},
     }
-    oemof_solph_object_asset = "object"
-    weighting_factor_energy_carrier = "weighting_factor_energy_carrier"
-    renewable_share_asset_flow = "renewable_share_asset_flow"
-    oemof_solph_object_bus = "oemof_solph_object_bus"
 
     (
         renewable_assets,
         non_renewable_assets,
     ) = D2.prepare_constraint_minimal_renewable_share(
         dict_values=dict_values,
         dict_model=dict_model,
-        oemof_solph_object_asset=oemof_solph_object_asset,
-        weighting_factor_energy_carrier=weighting_factor_energy_carrier,
-        renewable_share_asset_flow=renewable_share_asset_flow,
-        oemof_solph_object_bus=oemof_solph_object_bus,
+        oemof_solph_object_asset=OEMOF_SOLPH_OBJECT_ASSET,
+        weighting_factor_energy_carrier=WEIGHTING_FACTOR_ENERGY_CARRIER,
+        renewable_share_asset_flow=RENEWABLE_SHARE_ASSET_FLOW,
+        oemof_solph_object_bus=OEMOF_SOLPH_OBJECT_BUS,
     )
```

Comment:
Probably needs to be adapted if you consider my comments from above. I do not highlight this later anymore.